app/actions/electron.ts

```diff
@@ -1,24 +1,7 @@
-// @flow
-
-import { MAIN } from '../constants/targets';
-import type { Action } from '../reducers/types';
-import type { BrowserViewName } from '../reducers/electron';
-import {
-  ELECTRON_ROUTING,
-  HIDE_CONFIGURATION,
-  INTERNAL_ADD_BROWSER_VIEW,
-  PERFORM_SCROLL,
-  SET_BROWSER_VIEW_READY,
-  SET_BROWSER_VIEW_URL,
-  SET_BROWSER_WINDOW,
-  SET_INTERACTIVE_MODE,
-  SHOW_CONFIGURATION,
-  SHOW_DEV_TOOLS,
-  OPEN_PDF,
-  SET_AVAILABLE_VERSION,
-  SET_UPDATE_DOWNLOAD_PROGRESS
-} from '../constants/actionTypes';
-
+import { MAIN } from "../constants/targets";
+import type { Action } from "../reducers/types";
+import type { BrowserViewName } from "../reducers/electron";
+import { ELECTRON_ROUTING, HIDE_CONFIGURATION, INTERNAL_ADD_BROWSER_VIEW, PERFORM_SCROLL, SET_BROWSER_VIEW_READY, SET_BROWSER_VIEW_URL, SET_BROWSER_WINDOW, SET_INTERACTIVE_MODE, SHOW_CONFIGURATION, SHOW_DEV_TOOLS, OPEN_PDF, SET_AVAILABLE_VERSION, SET_UPDATE_DOWNLOAD_PROGRESS } from "../constants/actionTypes";
```

Comment:
is this super-long line the output of prettier?

Reply:
Nope, its from the translation tool.

I created issue #187 which should take care of correct styling

Reply:
#187 is good and important, but did you run prettier manually on the currently pushed code?

Reply:
I.e. I think we should think of #187 and having a pretty PR here as separate concerns.